_includes/footer/desktop.html

```diff
@@ -24,7 +24,7 @@
     </ul>
   </div>
   <div class="footer-links">
-    <h5 class="footer-links-heading">Policies and Standards</h5>
+    <h4 class="footer-links-heading">Policies and Standards</h4>
```

Comment:
I ran codesniffer and it recommended it be an h4. Is it better to have them be an h2?